app/components/Tags.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+/**
+ * List of tags displayed horizontally
+ *
+ * @param {{
+ *  tags: Array<String>,
+ *  style: Object,
+ *  tagStyle: Object,
+ *  fadeRight: boolean,
+ *  onClick: (e: Event) => void
+ * }} props
+ * @returns {React.Component}
+ */
+const Tags = React.forwardRef(
+  ({ tags, onClick, style, tagStyle, fadeRight }, ref) => {
+    return (
+      <ul
+        style={{
```

Comment:
lots of inline css in this file - consider moving to a separate file? like `'../styles/Tags.module.css'`

Reply:
I think it makes more sense here since we want our prop overrides to be in-scope